docs/changes.rst

```diff
@@ -31,6 +31,20 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in SubscriptionManager where 1. host_name  of
+  CIM_ListenerDestination and CIM_Filter objects inconsistently used
+  in building the string in the Name property and building the regex to
+  find the string. The result was that starting a subscription manager
+  against a server with owned subscriptions, etc. did not restore them to
+  the _owned... instance dictionaries, 2: _create_subscriptions did not
+  create complete path for new subscription before using it,
```

Comment:
Not sure we should mention internal methods in the change log. Describe it from the user's view.

Reply:
Changed to reference the external methods add_...  DONE